unit_tests/test_request_object.py

```diff
@@ -8,7 +8,7 @@ def test_request_object():
         path="/user",
     )
     request = Request(
-        query_params=QueryParams(),
+        query_params=RustQueryParams(),
```

Comment:
No, this should be QueryParams. No part of the client facing code should require us to initialise RustQueryParams